src/components/look-controls.js

```diff
@@ -147,11 +147,19 @@ module.exports.Component = registerComponent('look-controls', {
     hmdEuler.setFromQuaternion(hmdQuaternion, 'YXZ');
     if (isMobile) {
       // In mobile we allow camera rotation with touch events and sensors
-      rotation = {
-        x: radToDeg(hmdEuler.x) + radToDeg(pitchObject.rotation.x),
-        y: radToDeg(hmdEuler.y) + radToDeg(yawObject.rotation.y),
-        z: radToDeg(hmdEuler.z)
-      };
+      if (this.data.reverseMouseDrag) {
```

Comment:
It seems for mobile / touch, there might only be one expected direction since you're physically dragging it (drag the environment, not the camera). Since you wouldn't touch-drag while in first-person/VR.